packages/emails/src/templates/OrganizerScheduledEmail.tsx

```diff
@@ -32,6 +33,9 @@ export const OrganizerScheduledEmail = (
   const locale = props.teamMember?.language.locale || props.calEvent.organizer.language.locale;
   const timeFormat = props.teamMember?.timeFormat || props.calEvent.organizer?.timeFormat;
 
+  const isTeamEvent = props.calEvent.schedulingType === SchedulingType.ROUND_ROBIN || props.calEvent.schedulingType === SchedulingType.COLLECTIVE;
+  const attendee = isTeamEvent && props.teamMember ? props.teamMember : props.attendee;
```

Comment:
Not sure why but for Round Robin and Collective, suppose you had HOST A, B, and C, props.attendee was Host A even when sending emails for B and C.

Reply:
This change looks good 👍 It was wrong before, the email was in the team members' language but the text to cancel or reschedule the booking (`ManageLink`) was in the organizer's language